lighthouse/helpers/responses.py

```diff
@@ -4,11 +4,11 @@
 from lighthouse.types import FlaskResponse
 
 
-def bad_request(errors: Union[str, List[str]]) -> FlaskResponse:
+def bad_request(errors: Union[str, List[str]], **kwargs: Any) -> FlaskResponse:
```

Comment:
Do we want to be returning anything other than a list of errors?

Reply:
Sorry, this was something I added when I was writing the API myself before using Eve.  I'll undo this change.